extensions/hp-extension/src/hp/hpHeart.ts

```diff
@@ -52,25 +70,26 @@ export default {
               weight: 10,
               attribute: 'SeriesDescription',
               constraint: {
-                contains: '4Ch',
+                contains: '4',
```

Comment:
is it a safe check? just checking if SeriesDescription has "4" in it?

Reply:
I believe this sounds too client specific. Cant we have a way to configure the contains? or even the hp definition in total

Reply:
Yes, there is a plan to allow configuring the hanging protocol in the theme more generally, but I haven't quite gotten all the build setup working yet.